components/global/PlacesAndPolygons.vue

```diff
@@ -106,356 +105,266 @@ const getFlagEmoji = (countryCode: string) => {
+                  <td>-154.170</td>
+                  <td>0.8</td>
+                </tr>
+                <tr>
+                  <td>AK3</td>
+                  <td>Akiachak</td>
+                  <td>Akiacuar</td>
+                  <td>60.9094</td>
+                  <td>-161.431</td>
+                  <td>33.5</td>
+                </tr>
+                <tr>
+                  <td>AK4</td>
+                  <td>Akiak</td>
+                  <td>Akiaq</td>
+                  <td>60.9122</td>
+                  <td>-161.214</td>
+                  <td>43</td>
+                </tr>
+                <tr>
+                  <td>AK5</td>
+                  <td>Akutan</td>
+                  <td>Achan-ingiiga</td>
+                  <td>54.1385</td>
+                  <td>-165.778</td>
+                  <td>1.4</td>
+                </tr>
+                <tr>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
+                </tr>
+              </tbody>
+            </table>
           </div>
         </div>
-        <div class="is-hidden-desktop mb-6">&nbsp;</div>
       </div>
 
-      <div class="tile is-parent is-block-tablet is-flex-desktop">
-        <div class="tile is-child">
-          <div class="horizontal-divider mb-6"></div>
-          <div class="interactive-feature">
-            <div class="interactive-label">Interactive Feature</div>
-            <RandomPolygonMap />
-          </div>
-          <div class="is-hidden-desktop mb-6">&nbsp;</div>
-        </div>
-      </div>
-    </div>
+      <div class="column">
+        <div class="content is-size-5">
+          <h3 class="title is-4">Polygon Boundaries</h3>
 
-    <div class="tile is-ancestor is-size-5 is-block-tablet is-flex-desktop">
-      <div class="tile is-parent is-block-tablet is-flex-desktop">
-        <div class="tile is-child border-right is-12-tablet px-3">
-          <div class="horizontal-divider mb-6"></div>
-          <div class="mx-5">
-            <h4 class="title is-5">Point Location Features</h4>
-            <p>
-              Thousands of point locations are manually vetted for geographic
-              accuracy. Place name correctness is with third party governmental
-              organizations and research institutions.
-            </p>
-            <ul>
-              <li>Geographic extent: the circumpolar North</li>
-              <li>Decimal degree precision to 4 places (≈11m accuracy)</li>
-              <li>Pre-computed shortest distance between point and ocean</li>
-              <li>Multiple name variants where available</li>
-              <li>Unique alphanumeric ID for each point</li>
-            </ul>
+          <p>
+            Choose from over <strong>3,000 polygon features</strong> that enable
+            zonal statistics&mdash;cookie cutters, for data.
```

Comment:
HUC-12s are indeed important polygon features that enable zonal stats - so I think the prior number (order 17,000 or something like that) should be retained.